homeworkLecturesW10D03/src/ba/bitcamp/Files/MyPanel.java

```diff
@@ -0,0 +1,181 @@
+package ba.bitcamp.Files;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import javax.swing.JFileChooser;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.filechooser.FileNameExtensionFilter;
+
+/**
+ * @author Gordan
+ *
+ *         Making MyPanel that has next specifications: JTextArea used for
+ *         writing some text JMenuBar that contains JManu with tree JMenuItems:
+ *         New,Open and Save As JScrollPane with horizontal and vertical
+ *         scrollbars
+ *
+ *         Every JMenuItem has a specific ActionListener: New creates new empty
+ *         text area Open is used to open choosed text file Save As is used to
+ *         save text file at choosed location and under choosed name
+ */
+public class MyPanel extends JPanel {
+
+	private static final long serialVersionUID = -3466550005830151593L;
+
+	// Creating private objects used for developing MyPanel object
+
+	// Creating text area
+	private JTextArea textArea = new JTextArea();
```

Comment:
Variables should be instantiated in constructor rather than during declaration.

Both ways work, but, by my opinion, putting it in constructor should be the way to go (please take this as suggestion rather than something that must be done).